src/admin-views/editor/button-view-orders.php

```diff
@@ -1,11 +1,19 @@
 <?php
 $post_id = get_the_ID();
 
+// Get all the event providers.
+$providers = Tribe__Tickets__Tickets::get_active_providers_for_post( $post_id );
+
+// If RSVP is the only provider, bail out.
+if ( count( $providers ) == 1 && isset( $providers['Tribe__Tickets__RSVP'] ) ) {
```

Comment:
Let's have a yoda conditional and strict comparison for the first one.

Reply:
```suggestion
if ( 1 === count( $providers ) && isset( $providers['Tribe__Tickets__RSVP'] ) ) {
```